package.json

```diff
@@ -14,6 +14,7 @@
     "build:types": "tsc -p ./tsconfig.json --outDir build --declaration true && api-extractor run",
     "docs": "typedoc src --out docs",
     "test": "jest",
+    "test:auth": "jest -- auth.test.ts",
```

Comment:
You can actually just add any params you want onto the yarn function and they will get passed through. So for this you can just do: `yarn test auth` and it'll test only this file

Reply:
@nbragdon Cool. Just removed and tried that.